nservicebus/licensing/index_validity_core_[5,).partial.md

```diff
@@ -1,3 +1,3 @@
-The license is valid if the `ExpirationDate` or the `UpgradeProtectionExpiration` attribute in the license data is greater than or equal to the release date of the `{major}.{minor}.0` version of the NServiceBus assembly used by the endpoint. The release dates for the various versions can be found on [NuGet](https://www.nuget.org/packages/nservicebus).
+The license is valid if the `ExpirationDate` or the `UpgradeProtectionExpiration` attribute in the license data is greater than or equal to the release date of the `{major}.{minor}.0` version of the NServiceBus assembly used by the endpoint. To view the release dates for the various versions, see [NServiceBus Packages Versions](/nservicebus/upgrades/all-versions.md).
```

Comment:
> The license is valid if the `ExpirationDate` or the `UpgradeProtectionExpiration` attribute in the license data is greater than or equal to the release date of the `{major}.{minor}.0` version of the NServiceBus assembly used by the endpoint. 

I wonder if we should explain such things here, because:
1. The users shouldn't care about such low-level implementation details
2. Having that information they can easily modify the license ;) Aren't we provoking that behavior?

@andreasohlund thoughts ☝️